docs/README.md

```diff
@@ -1496,16 +1496,16 @@ mutation {
 }
 ```
 
-## Deploying
+### Serverless ready
 
 One of the goals of Booster is to become provider agnostic so you can deploy your application to any serverless provider like AWS, Google Cloud, Azure, etc...
 
 So far, in the current version, only AWS is supported, but given the high level of abstraction, it will eventually support
```

Comment:
Not sure if this paragraph is appropiate to be here, since we're currently having more versions (like Kubernetes now) almost ready.